packages/firebase_admob/lib/src/ad_instance_manager.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:quiver/collection.dart';
+
+import '../firebase_admob.dart';
+
+// ignore_for_file: public_member_api_docs
```

Comment:
Once the redesign is finished, this file will be hidden. It won't be available publicly.